packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -97,12 +105,19 @@ class UpgradeCommandRunner {
     @required GitTagVersion gitTagVersion,
     @required FlutterVersion flutterVersion,
     @required bool testFlow,
+    @required bool verifyOnly,
   }) async {
     final String upstreamRevision = await fetchRemoteRevision();
     if (flutterVersion.frameworkRevision == upstreamRevision) {
       globals.printStatus('Flutter is already up to date on channel ${flutterVersion.channel}');
       globals.printStatus('$flutterVersion');
       return;
+    }else if(verifyOnly){
```

Comment:
nit: there should be a space before `else`.

Reply:
Sure, fixing it now.